integration-tests/src/test/resources/coh/domaincreation.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+domainUID: cohdomain
+namespace: coh
+configuredManagedServerCount: 4
+initialManagedServerReplicas: 2
+exposeAdminT3Channel: true
+exposeAdminNodePort: true
+domainHomeImageBuildPath: ./docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt
+domainHomeImageBase: "store/oracle/weblogic:12.2.1.3"
+logHomeOnPV: true
+replaceDomainModelYaml: coh/coherence.yaml
```

Comment:
can you change the property name to domainModelYaml, give path as integration-tests/src/test/resources/coh/coherence.yaml and change the code accordingly? Otherwise looks good.

Reply:
Thanks @vanajamukkara! I will make that change and resubmit the PR with that change.